uwsgi.h

```diff
@@ -1039,6 +1039,8 @@ struct uwsgi_plugin {
 	void *data;
 	void (*on_load) (void);
 	int (*init) (void);
+	void (*pre_uwsgi_fork) (void);
+	void (*post_uwsgi_fork) (int);
```

Comment:
this doesn't cause any issues for other plugins?

Reply:
Nope, just as having `master_fixup` only used by the Python plugin didn't create a problem.